tests/test_io.py

```diff
@@ -1039,5 +1312,55 @@ def test_can_read_all_files_in_magicc6_dir(file_to_read):
         mdata.read(MAGICC6_DIR, file_to_read)
 
 
+@pytest.mark.parametrize("file_to_read", [f for f in TEST_OUT_FILES])
+def test_can_read_all_valid_files_in_magicc6_out_dir(file_to_read):
+    if file_to_read.endswith(("PARAMETERS.OUT")):
+        read_cfg_file(join(TEST_OUT_DIR, file_to_read))
+    else:
+        # Check if the file is on the list of invalid files
+        for p in invalid_out_files:
+            if re.match(p, file_to_read):
+                # Return early as this file is marked as ignored.
+                return
+
+        mdata = MAGICCData(file_to_read)
+        mdata.read(TEST_OUT_DIR)
+
+
+@pytest.mark.parametrize("file_to_read", [f for f in TEST_OUT_FILES])
+def test_cant_read_all_invalid_files_in_magicc6_out_dir(file_to_read):
+    # Check if the file is on the list of invalid files
+    valid_filename = True
+    for p in invalid_out_files:
+        if re.match(p, file_to_read):
+            valid_filename = False
+    # Skip files not marked as invalid
+    if valid_filename:
+        return
+
+    mdata = MAGICCData(file_to_read)
+    try:
+        with pytest.raises(NoReaderWriterError):
```

Comment:
For these files, raise a custom error message saying we're not planning on ever being able to read them because their format is so weird